src/Workspaces/Core/Portable/ExternalAccess/UnitTesting/Api/IUnitTestingIncrementalAnalyzerProvider.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
+{
+    internal interface IUnitTestingIncrementalAnalyzerProvider
```

Comment:
Use the `Implementation` suffix to indicate that this interface will be implemented in external code.

```suggestion
    internal interface IUnitTestingIncrementalAnalyzerProviderImplementation
```